src/helpers/notificationHandler.js

```diff
@@ -123,3 +123,33 @@ export const getPrivateChats = (io, staffs) => {
     });
   });
 };
+
+/** 
+ *  @param {Object} data
+ *  @override
+*/
+
+export const editRequestEmitter = data => {
+  tripEmitter.emit('edit Request', data)
+}
+
+/**
+ * Function to listen for the created comment event and send
+ * @param {Object} io
+ * @param {Object} staffs
+ * @override
+ */
+export const requestUpdateNotification = async  (io, staffs) =>{
+
+  tripEmitter.on('edit Request', async (payload) => {
+    
+    const [ Roles ] = await getAllManager();
+    const managers = Roles.toJSON();
+    const { users } = managers;
+    const managersEmail = users.map(items => items.email);
+    const notification = await createNotification(payload);
+    managersEmail.forEach(email => {
+      io.to(staffs[email]).emit('edit Request', notification);
+    });
+  })
```

Comment:
Rather than this
```
  managersEmail.forEach(email => {
      io.to(staffs[email]).emit('edit Request', notification);
    });
```

,we can do this
```
users.forEach(({ email }) => {
      io.to(staffs[email]).emit('editRequest', notification);
    });
```
instead to reduce the number of overall iteration we have above `const managersEmail = users.map(items => items.email);`